lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -264,6 +299,64 @@ void fetchAccessToken_whenCalledAgainstPassportCri_retrievesAValidAccessToken(
         assertThat(accessToken.getLifetime(), greaterThan(0L));
     }
 
+    @Test
+    @PactTestFor(pactMethod = "invalidAccessTokenRequestReturnsError")
+    void fetchAccessToken_whenCalledAgainstPassportCri_retrievesAnInValidAccessToken(
```

Comment:
`InValid` -> `Invalid`

Reply:
Also this test name doesn't match what you are expecting. The provider isn't returning an invalid access token, you're expecting it to return an error code.